common/tools/dev-tool/src/commands/run/extract-api.ts

```diff
@@ -33,275 +32,274 @@ export const commandInfo = makeCommandInfo(
-    log.debug(`API Extractor completed successfully`);
-    return true;
+  if (result.succeeded) {
+    log.debug("API Extractor completed successfully");
   } else {
     log.error(
-      `API Extractor completed with ${extractorResult.errorCount} errors` +
-        ` and ${extractorResult.warningCount} warnings`,
+      `API Extractor completed with ${result.errorCount} errors and ${result.warningCount} warnings`,
     );
-    return false;
   }
+  return result.succeeded;
 }
 
-export default leafCommand(commandInfo, async (options) => {
-  const projectInfo = await resolveProject(process.cwd());
-  const packageJsonPath = path.join(projectInfo.path, "package.json");
-  const packageJson = JSON.parse(await readFile(packageJsonPath, { encoding: "utf-8" }));
-
-  const reviewDirPath = path.join(projectInfo.path, "review");
-  if (!existsSync(reviewDirPath)) {
-    log.info(`creating directory ${reviewDirPath}`);
-    await mkdir(reviewDirPath);
+// TODO: explore better diff algorithms that consider changes at the definition level instead of the line level
+function createApiDiff(
+  nodeContent: string,
+  runtimeContent: string,
+  runtime: string,
+): string | undefined {
+  const nodeLines = nodeContent.split("\n");
+  const runtimeLines = runtimeContent.split("\n");
+  const diff: string[] = [];
+  const maxLength = Math.max(nodeLines.length, runtimeLines.length);
+  let hasChanges = false;
+  for (let i = 0; i < maxLength; i++) {
+    const nodeLine = nodeLines[i] || "";
+    const runtimeLine = runtimeLines[i] || "";
+    if (nodeLine !== runtimeLine) {
+      hasChanges = true;
+      diff.push(`- ${nodeLine}`);
+      diff.push(`+ ${runtimeLine}`);
+    }
   }
-
-  const apiExtractorJsonPath: string = path.join(projectInfo.path, "api-extractor.json");
-  const extractorConfigObject = ExtractorConfig.loadFile(apiExtractorJsonPath);
-  // sub path exports extraction
-  const exports = buildExportConfiguration(packageJson);
-  if (
-    !extractorConfigObject.mainEntryPointFilePath ||
-    (exports === undefined && !extractorConfigObject?.dtsRollup?.publicTrimmedFilePath)
-  ) {
-    log.error("Unexpected api-extractor configuration");
-    return false;
+  if (!hasChanges) {
+    return undefined;
   }
+  const preamble = `# API Report Diff for ${runtime} runtime
 
-  log.debug(`  mainEntryPointFilePath: ${extractorConfigObject.mainEntryPointFilePath}`);
-  log.debug(`  publicTrimmedFilePath: ${extractorConfigObject?.dtsRollup?.publicTrimmedFilePath}`);
-  log.debug(`  apiJsonFilePath: ${extractorConfigObject.docModel?.apiJsonFilePath}`);
-  log.debug(`  reportFileName: ${extractorConfigObject.apiReport?.reportFileName}`);
-  log.debug(`  reportTempFolder: ${extractorConfigObject.apiReport?.reportTempFolder}`);
-
-  let succeed = true;
+This file contains only the differences from the Node.js API.
+For the complete API surface, see the corresponding -node.api.md file.
+`;
+  return [preamble, `\`\`\``, ...diff, `\`\`\``].join("\n");
```

Comment:
Could we add a language label here so that GitHub/VSCode gives some syntax highlighting for the diff?

```suggestion
  return [preamble, `\`\`\`diff`, ...diff, `\`\`\``].join("\n");
```